thefuck/rules/choco_install.py

```diff
@@ -0,0 +1,29 @@
+def match(command):
+    return (('choco install' in command.script_parts or 'cinst' in command.script_parts)
+            and 'Installing the following packages' in command.output)
+
+
+def get_new_command(command):
+    cmdList = command.script.split(' ')
+    packageName = ""
+    # Find the argument that is the package name
+    for i in cmdList:
+        print(i)
+        if "choco" in i:
```

Comment:
This `if` statement sequence should be rewritten with all expressions grouped together with `and` operators.